gui/src/components/ProcessList.svelte

```diff
@@ -61,14 +61,24 @@
     loadInitialLogs(workspaceId, workspace);
   }
 
+  let procfileExport: string;
```

Comment:
If you use:

```suggestion
let procfileExport: string | null = null
```

Then below, you won't need the `undefined as any` cast.